.github/workflows/01-workflow.yaml

```diff
@@ -9,6 +9,10 @@ on:
       - main
 
 jobs:
+  env:
```

Comment:
env should be in "root" property i think 
```yaml
env:
   ... 
jobs: 
  ...
```

Reply:
Thanks :)